src/Shared/Util/Math2.cs

```diff
@@ -70,5 +73,45 @@ public static bool Between(int val, int min, int max)
 		{
 			return (val >= min && val <= max);
 		}
+
+		/// <summary>
+		/// Calculate rotation toward target
+		/// </summary>
+		/// <param name="source"></param>
+		/// <param name="target"></param>
+		/// <returns></returns>
+		public static Direction AngleBetweenTwoEntity(Position source, Position target)
```

Comment:
I'd propose to move this to Position or Direction, as it's not a general use Math function, but something very specific to the game. Same for VectorTwoEntity.